addon/components/labeled-radio-button.js

```diff
@@ -8,10 +8,16 @@ export default Component.extend({
   tagName: 'label',
   layout,
   attributeBindings: ['for'],
-  classNameBindings: ['checked'],
+  classNameBindings: ['_checkedClass'],
   classNames: ['ember-radio-button'],
   defaultLayout: null, // ie8 support
 
+  checkedClass: 'checked',
+  _checkedClass: computed('checked', 'checkedClass', {
+    get() {
```

Comment:
it looks like the ember 1.11 test case is failing because of this. travis output contains `Error: Computed Property declared without a property function`